.gitignore

```diff
@@ -12,3 +12,5 @@ runtime-EclipseApplication-m2eclipse
 ./fuseenterprise
 .idea
 libs/
+
```

Comment:
Yeah, but I always put an empty line at the bottom. Its not really crucial to remove this line I'd say.